docs/dataformats/observation_lists.rst

```diff
@@ -22,23 +22,48 @@ This has the advantage that everyone can work with whatever tool they like. Here
 * `csvkit <https://csvkit.readthedocs.org/en/latest/>`_
 * Your own script using e.g the `Python standard library CSV module <http://docs.python.org/2/library/csv.html>`_ or `pandas <http://pandas.pydata.org>`_
 
-A run list must have at least a column called ``Run``::
+A run list must have at least a column called ``OBS_ID``::
  
-   Run
+   OBS_ID
    42
    43
 
-Usually it has many more columns with information about each run::
- 
-   Run,Telescope_Pattern,Date,Duration,GLON,GLAT,Target,Zenith
-   1234,24,2013-03-22 14:32,1832,83.7,-5.2,Crab Nebula,32
+Usually it has many more columns with information about each observation. A list of Gammapy supported columns is::
+
+   OBS_ID: observation ID as an integer (starting at 0 or 1? does it matter?)
+   RA: pointing position right ascension in equatorial (ICRS) coordinates
+   DEC: pointing position declination in equatorial (ICRS) coordinates
+   AZ: average azimuth angle during the observarion
+   ALT: average altitude angle during the observarion
+   MUON_EFFICIENCY: average muon efficiency of the telescopes
+   TIME_START: start time of the observation stored as number of seconds after the time reference in the header
+   TIME_STOP: end time of the observation in the same forma as TIME_START
+   TIME_OBSERVATION: duration of the observation
+   TIME_LIVE: duration of the observation without dead time
+   TRIGGER_RATE: average trigger rate of the system
```

Comment:
I find it difficult to label the columns as required or optional.
Actually, we already said before that the only required column is `OBS_ID`.
I guess it depends on what do you want to do with the table afterwards. If you want to run an analysis, the only column needed is the `OBS_ID`: gammapy should be able to find the corresponding observation lists.
If you want to make a selection of runs (apply some filter, like findruns does), you need the columns that you want to filter on.
Plus, chances are that we label a column as optional, then implement some tool that works on it and forget to update the doc.